frontends/bmd/src/app_root.tsx

```diff
@@ -1181,17 +1132,13 @@ export function AppRoot({
     void updateStorage();
     startCardShortValueReadPolling();
     startLongValueWritePolling();
-    startHardwareStatusPolling();
     return /* istanbul ignore next - triggering keystrokes issue - https://github.com/votingworks/bmd/issues/62 */ () => {
       stopCardShortValueReadPolling();
-      stopHardwareStatusPolling();
```

Comment:
I'm not sure if there was any particular reason to start/stop battery status polling here in particular. It seems like it should be ok to just do it in a hook on mount, but if there was a reason to do it here, it would be good to think about.